tvb_library/Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+numba = "*"
+pytest = "*"
+matplotlib = "*"
+scipy = "*"
+mako = "*"
+autopep8 = "*"
+tvb-gdist = "*"
+pylems = "*"
+h5py = "*"
+
+[dev-packages]
+pytest = "*"
+pytest-cov = "*"
+pytest-xdist = "*"
+pytest-benchmark = "*"
+pytest-mock = "*"
+
+[requires]
+python_version = "3.9"
```

Comment:
If I am not mistaken EBRAINS lab is Python 3.8, maybe test against that with priority?